src/components/user/repo/user.repo.js

```diff
@@ -9,6 +9,18 @@ const getUser = async (_id) => {
 	return user;
 };
 
+const createTombstoneUser = async (schoolId, tombstoneSchoolId) => {
+	// no school id so this user does not come up in find schools users
+	const user = {
+		email: `tombstone-${schoolId.toString()}@hpi-schul-cloud.org`,
+		firstName: 'Gelöschter',
+		lastName: 'Benutzer',
+		schoolId: tombstoneSchoolId,
+	};
+	const createdUser = await User.create(user);
```

Comment:
please check if zou could use lean(). exec() instead of toObject()

Reply:
Model.create() doesn't return a query object and Model.save() returns a query object only on updates. So I suppose this is not possible:
https://mongoosejs.com/docs/api/model.html#model_Model.create
https://mongoosejs.com/docs/api/model.html#model_Model-save